src/Shell/Task/ModelTask.php

```diff
@@ -104,37 +104,51 @@ public function main($name = null)
      */
     public function bake($name)
     {
+        extract($this->getTableObjectSetup($name)); // $tableObject, $data
+        $this->bakeTable($tableObject, $data);
+        $this->bakeEntity($tableObject, $data);
+        $this->bakeFixture($tableObject->alias(), $tableObject->table());
+        $this->bakeTest($tableObject->alias());
+    }
+
+    /**
+     * Get table object setup for bake
+     *
+     * @param string $name The model name to generate.
+     * @return array
+     */
+    public function getTableObjectSetup($name)
```

Comment:
I needed to change/add this to have an easy time callig bakeTable() setup like bake itself does (instead of faking it like the other tests do).